.spectral.yaml

```diff
@@ -0,0 +1,202 @@
+# https://meta.stoplight.io/docs/spectral/cb95cf0d26b83-core-functions
```

Comment:
[q] can we have a follow-up task to import these in mms? would be great to have a single point of rules!

Reply:
sure, it makes sense 👍

Reply:
https://jira.mongodb.org/browse/CLOUDP-263187

Reply:
Consider adding Contributing guide. Otherwise upstream PR with new rules can break number of MMS PRs. 
We seeing similar challenges in the MMS validator.

As alternative we can consider backend rules to be source of truth and now (manually) contribute spectral rules upstream.